app/controllers/discourse_ai/admin/ai_personas_controller.rb

```diff
@@ -32,6 +36,11 @@ def create
         end
       end
 
+      def create_user
```

Comment:
Probably a good idea to rate limit this. Less dangerous since it's admin-only, but still.

Reply:
hmmm yeah I am not sure actually, you could just create personas non stop anyway, maybe default rate limiting is enough.

Reply:
Ah yeah true I forget about the defaults